ui/static/ui/js/utils.jsx

```diff
@@ -116,6 +116,109 @@ define("utils", ["jquery", "lodash", "react", "react_infinite", "select2"],
   });
 
   /**
+  * Callback to send confirmation status.
+  *
+  * @callback confirmationResponse - Call back that sends confirmation status
+  * @param {bool} status - Confirmation status
+  */
+
+  /**
+   * Generic confirmation dialog.
+   * <ConfirmationDialog
```

Comment:
You can leave out the React element syntax and just describe the props since it's self evident how to pass props to the element